.gitignore

```diff
@@ -1,6 +1,6 @@
-docs/
 reports/
-var/
```

Comment:
Although `docs/` can be dropped (unless you want the jsdoc files bundled that I added later in the Promises PR), `var` should I think be excluded, as the `view-cov` script ends up adding a `var` directory (through our Makefile's inclusion of `make-node`'s Make files).

Reply:
I also think we need `.github` added here.